eng/BranchInfo.props

```diff
@@ -2,30 +2,41 @@
   <PropertyGroup>
     <StableProjects>
       Microsoft.Extensions.ML;
-      Microsoft.ML.DataView;
-      Microsoft.ML.CpuMath;
       Microsoft.ML;
       Microsoft.ML.Core;
+      Microsoft.ML.CpuMath;
       Microsoft.ML.Data;
-      Microsoft.ML.KMeansClustering;
-      Microsoft.ML.PCA;
-      Microsoft.ML.StandardTrainers;
-      Microsoft.ML.Transforms;
+      Microsoft.ML.DataView;
       Microsoft.ML.FastTree;
       Microsoft.ML.ImageAnalytics;
+      Microsoft.ML.KMeansClustering;
       Microsoft.ML.LightGbm;
       Microsoft.ML.Mkl.Components;
       Microsoft.ML.Mkl.Redist;
-      Microsoft.ML.TimeSeries;
-      Microsoft.ML.TensorFlow;
       Microsoft.ML.OnnxTransformer;
+      Microsoft.ML.PCA;
+      Microsoft.ML.StandardTrainers;
+      Microsoft.ML.TensorFlow;
+      Microsoft.ML.TimeSeries;
+      Microsoft.ML.Transforms;
       Microsoft.ML.Vision;
     </StableProjects>
+
+    <StableLowerVersionProjects>
```

Comment:
Is there a way we can avoid the static list and instead make this opt-in?  Perhaps by setting a property in the project's Directory.Build.props or the Project body?

Reply:
I'll try to do that

Reply:
I found that `<Import Project="eng/BranchInfo.props" />` is done early in `Directory.Build.props`. Setting the property in csproj will not work. Should I move the logic I added in `BranchInfo.props` to `Directory.Build.props`? This will make changing the version logic split to two places. The other idea is to move all version logic out of BranchInfo.props (bigger change).